[milestone/11.5.2] Milestone 11.5.2

hosy opened the pull request:
## Description
PR for upcoming version 11.5.2

## Related Issue
#914 

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->

## How Has This Been Tested?
<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->

## Screenshots (if appropriate):

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING**](https://github.com/owncloud/ios-app/blob/master/CONTRIBUTING.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] Added changelog files for the fixed issues in folder changelog/unreleased

hosy committed:
Preparing new version: changed version and build number to 11.5.2 and 188

felix-schwarz committed:
Add Themeable support to the Acknowledgements (#900)

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/owncloud/ios-app?pullRequest=901) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/owncloud/ios-app?pullRequest=901) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: hosy<br/>:x: felix-schwarz<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/owncloud/ios-app?pullRequest=901) it.</sub>

felix-schwarz committed:
[feature/flat-mdm-settings] Support for MDM setting hierarchies with flat keys (#904)

* - automatic termination + relaunch by notification on MDM changes push
- fix indentation

* - react to setting changes pushed via MDM differently, in a two-step process
	- first notify the user about the changes
	- if the user taps the notification, schedule a relaunch notification and then quit
	- if the user taps the relaunch notification, the app is launched
- request authorization to post notifications if needed

Commit:
[fix/pdf-thumbnailview-position] PDF thumbnail view position on the iPad (#905)

* Fixing the thumbnail view position on the iPad in portrait view
- added orientation changed notification to recreate the constrains, when the device was rotated

* added changelog issue

* Calens changelog updated

* fixed QA finding (jumpy thumbnail view on iPhone)

* isPortrait mode is not working and removed it with size class

Co-authored-by: hosy <hosy@users.noreply.github.com>

felix-schwarz committed:
[fix/collapsible-progress-bar-position-flex] Collapse Progress Bar in Detail View (#911)

mneuwert committed:
Accessing hyper links in PDF view (#913)

hosy committed:
Updated release notes for version 11.5.2

hosy committed:
Calens changelog updated

Commit:
[feature/owncloud-online-screentshots] Create Screenshots for Branded Builds with Fastlane (#912)

hosy committed:
updated In-App release notes with marketing fix

hosy committed:
moved calens changelog files into release folder

hosy committed:
Calens changelog updated

hosy committed:
updated SDK to latest master commit
